### PR TITLE
Make it easier to create new envfile tests.

### PR DESCRIPTION
**Pull request by nirs**
- Clean tests for missing addons
- Make it easier to create new envfile tests

### CODE REVIEW

**Comment by nirs** [comment]
@Shwetha-Acharya can you review?

**Review by Shwetha-Acharya — APPROVED** [review]

File: test/drenv/envfile_test.py [hunk 1]
```diff
@@ -186,6 +181,32 @@ def test_name_prefix(tmpdir):
     assert worker["addons"][0]["args"] == []
 
 
+def test_missing_profile_addons(tmpdir):
+    s = """
+name: missing
```
Comment (Shwetha-Acharya) [inline test/drenv/envfile_test.py:192]: this word "missing" looks incomplete, better to name it test or missing-profile-addons, similar to other minimal yamls?

Comment (nirs) [reply test/drenv/envfile_test.py:192]: I'll use "test" like the other tests for consistently. Repeating the name of the test in the yaml does not make sense.

**Comment by raghavendra-talur** [comment]
I did a review but need ack from someone who knows Python better to merge this, @rakeshgm please review.

**Review by Shwetha-Acharya — APPROVED** [review]

**Pull request merged**